Packs/CommonScripts/Scripts/UnzipFile/UnzipFile_test.py

```diff
@@ -46,21 +46,25 @@ def test_unzip_no_password(file_name):
 
 
 data_test_unzip_with_password = [
-    ('fix_unzip.png', 'demisto'),
+    ('fix_unzip.png', 'demisto', '7z'),
+    ('fix_unzip.png', 'demisto', 'zipfile')
```

Comment:
instead of duplicating this line, use a second `@pytest.mark.parametrize` for the two zip_tools